app/grandchallenge/evaluation/templates/evaluation/submission_form.html

```diff
@@ -25,10 +25,15 @@
         {% if "change_challenge" in challenge_perms or user_is_participant %}
             {% for phase in challenge.phase_set.all %}
                 <li class="nav-item">
-                    <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' and request.resolver_match.kwargs.slug == phase.slug %}active{% endif %}"
+                    <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' and request.resolver_match.kwargs.slug == phase.slug or request.resolver_match.view_name == 'evaluation:submission-create-legacy' and request.resolver_match.kwargs.slug == phase.slug%}active{% endif %}"
```

Comment:
Is is right to include `evaluation:submission-create-legacy` here?

Reply:
I included it so that the respective phase tab would be highlighted when creating a submission for someone else. Looked nicer to me, but it's not strictly necessary, of course. 
![image](https://user-images.githubusercontent.com/30069334/113261562-e6067300-92cf-11eb-8818-bcc68fa9a1b6.png)
If we don't include it, it looks like this:
![image](https://user-images.githubusercontent.com/30069334/113261673-08988c00-92d0-11eb-906f-ede3fdb5f86b.png)

Reply:
I agree, this is great! Just wasn't sure where it was being linked from.